tests/Makefile

```diff
@@ -235,8 +266,9 @@ else
 endif
 
 # Test suites caught by SKIP_TEST_SUITES are built but not executed.
-check: $(BINARIES)
+check: $(BINARIES) $(CRYPTO_BINARIES)
 	perl scripts/run-test-suites.pl $(TEST_FLAGS) --skip=$(SKIP_TEST_SUITES)
+	cd ../tf-psa-crypto/tests && perl ../../tests/scripts/run-test-suites.pl $(TEST_FLAGS) --skip=$(SKIP_TEST_SUITES)
```

Comment:
[Bug: the crypto tests are skipped if a TLS test fails](https://github.com/Mbed-TLS/mbedtls/pull/9557).

Why doesn't `run-test-suites.pl` take care of looking for the subproject's test suites? (Like `ctest` does.)

Reply:
>  Why doesn't `run-test-suites.pl` take care of looking for the subproject's test suites?

https://github.com/Mbed-TLS/mbedtls/pull/9558